src/GNSS.cpp

```diff
@@ -4,6 +4,9 @@
 #include <MicroNMEA.h>
 #include "GNSS.h"
 
+
+#define pi 3.14159265358979323846
```

Comment:
Macro definitions - use capital letters

Reply:
Removed altogether in favour of builtin M_PI